src/main/java/graphql/nadel/result/ResultComplexityAggregator.java

```diff
@@ -0,0 +1,46 @@
+package graphql.nadel.result;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class ResultComplexityAggregator {
+    private Integer totalNodeCount;
```

Comment:
Use `int` whenever you can - the count cant be null so there is no need for `Integer`